containerized_ml_models/model_api_7/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM continuumio/anaconda3
+
+COPY app.py model.pkl requirements.txt /
+
+RUN conda create --name env --file requirements.txt
+
+ENTRYPOINT [ "conda", "run", "-n", "env" ,"--no-capture-output", "python", "app.py"]
```

Comment:
You should consider adding new line at the end od the file.
Explanation: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline